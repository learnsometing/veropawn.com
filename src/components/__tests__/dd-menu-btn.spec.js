/*
* Tests for the Dropdown Menu component.
*/

import React from "react";
import renderer from "react-test-renderer";

import DDMenuBtn from "../dd-menu-btn";

describe("Dropdown menu button", () => {
  test("Matches the snapshot", () => {
    const ddMenuBtn = renderer.create(<DDMenuBtn children={'foo'} />);
    expect(ddMenuBtn.toJSON()).toMatchSnapshot();
  });
});