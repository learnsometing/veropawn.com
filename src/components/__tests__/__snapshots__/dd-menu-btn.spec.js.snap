// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dropdown menu button Matches the snapshot 1`] = `
<li
  className="listItem"
>
  <button
    className="link"
  >
    foo
  </button>
</li>
`;
